src/main/java/seedu/address/model/person/Age.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's age recorded in the app.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAge(String)}
+ */
+
+public class Age {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Age should only contain numbers.";
+
+    public static final String VALIDATION_REGEX = "\\d{1,3}";
+    public final int value;
+
+    /**
+     * Constructs a {@code Age}.
+     * @param age A valid age.
+     */
+    public Age(String age) {
+        requireNonNull(age);
+        checkArgument(isValidAge(age), MESSAGE_CONSTRAINTS);
+        value = Integer.parseInt(age);
+    }
+
+    /**
+     * Returns true if a given string is a valid age.
+     */
+    public static boolean isValidAge(String test) {
```

Comment:
i'm not very sure, does the java doc require the parameter and the return value? eg.
```
/**
 * Returns true if a given string is a valid age.
 *
 * @param test....
 * @return boolean.....
**/
```

Reply:
I followed their existing format. :P Since checkstyle doesn't give error, I assume it's fine?

Reply:
okay can!